readme.md

```diff
@@ -24,7 +24,6 @@ npm run update-buildsystem
 ### Development loop
 
 ```
-npm run watch &
```

Comment:
we-ext can auto reload from changes.

Reply:
It can, but excmds_macros.py won't be re-run and the typescript won't be recompiled.

So this line watches the development files and rebuilds, and the next line watches the built files and reloads the extension as required.

Reply:
Ah, yeah, you are right, will revert this.